src/main/java/dal/fatih/todoly/Main.java

```diff
@@ -46,67 +45,51 @@ public static Task handleCreateTask() {
         }
         return handleCreateTask();
     }
-
     public static void listAllTasks() {
         if (tasks.isEmpty()) {
             System.out.println("Task list is empty");
         } else {
-            for (Task task : tasks) {
-                System.out.println("ID: " + task.getId() + " TITLE: " + task.getTitle() +
-                        " DUE DATE: " + dateFormatter(task.getDate()));
+            for (Map.Entry task : tasks.entrySet()) {
+                System.out.println("KEY: " + task.getKey() + task.getValue());

```

Comment:
Don't print key, it is confusing to see Task ID twice since task id will already be printed by toString method